src/Components/NavAside/NavAside.jsx

```diff
@@ -1,6 +1,15 @@
 import "./NavAside.css";
-
+import { useNavigate } from "react-router-dom";
+import { useAuth } from "../../Context/AuthContext";
 export default function NavAside({ navToggle, asideToggleFunction, setNavToggle }) {
+    const {
+        authState: { isAuth },
+    } = useAuth();
+    const navigate = useNavigate();
+    function asideNavigate(navigateTo) {
+        navigate(navigateTo);
+        asideToggleFunction();
+    }
```

Comment:
I would suggest you to give a new line after a function ends, or while starting a new function as it looks much cleaner to read
like
```
const a = 5;

const func = () => {
}

const b = 5 
```